views/agency-requests/agency_requests.php

```diff
@@ -16,13 +16,41 @@
             <br />
             <form class="search-agency" method="POST" action="/agency-requests">
                 <div class="form-group">
-                    <input type="text" class="form-control" name="searchquery" placeholder="Begin typing participant's name...">
-                </div>
-                <button type="submit" class="btn cpca form-control">Submit</button>
+                    <input type="text" class="form-control user-search" name="searchquery" placeholder="Begin typing participant's name...">
```

Comment:
Add comment header to the top of this PHP file (not where this comment is, GitHub wouldn't let me make a comment at the top of the file).